app/App_Resources/Android/app.gradle

```diff
@@ -18,6 +18,7 @@ android {
   
   }  
   aaptOptions {  
-    additionalParameters "--no-version-vectors"  
+    additionalParameters "--no-version-vectors"
```

Comment:
there were some extra spaces here, it seems. So i removed those also, wont cause any harm i guess :)